swift/ql/src/diagnostics/internal/CompilerErrors.ql

```diff
@@ -0,0 +1,11 @@
+/**
+ * @name Compiler errors
+ * @description List all compiler errors for files in the source code directory.
+ * @kind diagnostic
+ * @id swift/diagnostics/compiler-errors
```

Comment:
Can do either way with the id, though I thought about it a bit and not sure what's the right option here: on one hand, there are not "extraction errors" such as e.g. "cannot extract something because of our wrong assumption about the AST structure". On the other hand, these _are_ extraction errors as they are coming from the extractor (e.g. "the extractor hit a compiler error because we are trying to extract a new language feature that is not supported yet").

Anyways, I think it makes sense to stick to the common naming to avoid discrepancy 😅

Reply:
@MathiasVP should I also change the file name as well? 🤔

Reply:
Hm, yeah maybe that's a good idea to change the name as well. It seems like we're less consistent about the file name, though 😂.